x/circuit/go.mod

```diff
@@ -1,101 +1,151 @@
-module github.com/cosmos/cosmos-sdk/x/circuit
+module cosmossdk.io/x/circuit
 
 go 1.20
 
 require (
+	cosmossdk.io/api v0.3.1
+	cosmossdk.io/core v0.6.1-0.20230309163709-87da587416ba
+	cosmossdk.io/depinject v1.0.0-alpha.3
+	cosmossdk.io/errors v1.0.0-beta.7
+	cosmossdk.io/math v1.0.0-rc.0
 	cosmossdk.io/store v0.1.0-alpha.1
-	github.com/cosmos/cosmos-sdk v0.46.0-beta2.0.20230314215525-ba8de97d1af8
+	github.com/cockroachdb/errors v1.9.1
+	github.com/cometbft/cometbft v0.37.0
+	github.com/cosmos/cosmos-sdk v0.47.0
```

Comment:
It should be a pseudo version from main and not v0.47.0. It isn't compatible with simapp otherwise.